pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/cleanup/index.md

```diff
@@ -45,16 +46,15 @@ kubectl delete -n istio-system secret sh.helm.release.v1.istio-kubeaddons.v1
 
 If Gatekeeper is installed on the cluster, the upgrade leaves a Helm Secret, Deployments, and a Service that belonged to the 1.8 Gatekeeper addon in the `kubeaddons` namespace.
 
-<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, `helm list -n kubeaddons` incorrectly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
+<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, <code>helm list -n kubeaddons</code> incorrectly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, <code>helm list -n kubeaddons</code> incorrectly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper Platform Application in Kommander, making Kubernetes inoperable on the cluster.</p>
```